### PR TITLE
Edit README.md

### PR DESCRIPTION
**Pull request by qkrybin**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [x] others (anything other than above)

---

**Related:** #354 

**Summary**

- update README.md

---

After Update 

image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,56 +2,80 @@
 
 <br/>  
 <div align="center" style="display:flex;">  
-  <img width="600" src="https://user-images.githubusercontent.com/6308023/165422882-2ab14b87-7370-4e3a-9a01-94b545a26316.png">
+  <img width="400" alt="v_black_square_800px" src="https://user-images.githubusercontent.com/63234878/209922215-cccf7b88-de12-42ac-9714-8a5e3f194d7f.png">
```
Comment (xellos00) [inline README.md:5]: Thanks for updating a pictures. Appreciate it!!

File: README.md [hunk 2]
```diff
@@ -2,56 +2,80 @@
...
-[<img alt="Twitter" src="https://user-images.githubusercontent.com/6308023/176984455-d48b24a9-1eb4-4c38-b728-2f4a0ccff09b.svg" width="50" height="50" />](https://twitter.com/dsrvlabs)
+  ### Multi Protocol Support
+   **VATZ** is NOT limited Protocol Type where it categorizes on chain protocol. Any Protocol can be managed through VATZ with plugins, even unsupported protocols can be integrated through simple plugin development.
+  ### Infrastructure as Code
+  **VATZ** is described using a high-level configuration syntax. You can divide your plugins into modular components that can then be combined in different ways to behave through automation.
+  ### Data Analysis
+   **VATZ** helps to build datasets for your managing protocols and transfer your data into popular services Prometheus, Kafaka, Google BigQuery and more. Because of this, VATZ aims to set your Node infrastructure as efficiently as possible, and operators get insight into dependencies in their infrastructure. (2023-Q3)
+  ### Change automation
+  Complex sets of node's operational tasks can be done through **VATZ** with minimal human interaction. (2023-Q2)
```
Comment (xellos00) [inline README.md:42]: Can you push this to Q4

File: README.md [hunk 3]
```diff
@@ -2,56 +2,80 @@
...
 
+## How does **VATZ Project** work?
 
-#### _To find more about_ **`VATZ Project`**
-- [VATZ Project Design](docs/design.md)
+VATZ project is primarily designed to check the node states in real-time and receive alert notifications of all blockchain protocols, including metrics that the protocol itself does not support. 
 
-#### _To get start with_ **VATZ**
-- [Installation guide](docs/installation.md)
+To this end, it consists of 3 components:
```
Comment (xellos00) [inline README.md:25]: @qkrybin 
Can you add here?
#### **VATZ** _CLI Command list_

@rootwarp 
Can you put the link of CLI command md.file here once you create a file.

File: README.md (hunk 2 above)
Comment (qkrybin) [reply README.md:42]: data analysis and automation both?

File: README.md [hunk 4]
```diff
@@ -2,56 +2,80 @@
...
 
-## 2. Our Mission ##
+1. VATZ : VATZ executes plugin APIs based on configs, checks the health of plugins, and sends notifications to configured channels
+2. Plugins (SDK): Various features like checking node status, collecting node metrics and executing certain commands, can be integrated to VATZ through separate plugins. 
+3. Monitoring: Various logs and data of nodes are exported by a node exporter and monitored through the 3rd party applications like Graphana.
```
[the remark this answers is not in the document]
Comment (qkrybin) [reply README.md]: sorry for the typo, I just fixed it!

File: README.md (hunk 3 above)
Comment (qkrybin) [reply README.md:25]: @xellos00 
Can I add the command list in the 'usage' part?

Comment (xellos00) [reply README.md:25]: > @xellos00 Can I add the command list in the 'usage' part?

Ah, Yeah. you are right. 
It should be in the 'usage' part.

File: README.md (hunk 2 above)
Comment (xellos00) [reply README.md]: Data Analysis -> 2023 Q3
Change automation -> 2023 Q4

**Review by xellos00 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -2,56 +2,80 @@
...
     <a href="https://www.apache.org/licenses/LICENSE-2.0"  target="_blank"><img alt="License: Apache 2.0"  src="https://img.shields.io/badge/License-Apache 2.0-yellow.svg" /></a> 
   </p> 
-</div>    
+</div> 
 
-## 1. About VATZ ##
+# What is VATZ?
 
-> **VATZ** is a tool for building, analyzing, and managing blockchain Node infrastructure safely and efficiently. You can set up VATZ to manage existing or new blockchain nodes and integrate with popular services PagerDuty, Discord, and more as well as custom in-house solutions.
+**VATZ** is a tool for building, analyzing, and managing blockchain node infrastructure safely and efficiently. You can set up VATZ to manage existing or new blockchain nodes and integrate with popular services PagerDuty, Discord, and more as well as custom in-house solutions.
```
Comment (skonhwang) [inline README.md:16]: @qkrybin , How about add telegram?

Comment (qkrybin) [reply README.md]: added!

File: README.md (hunk 2 above)
Comment (qkrybin) [reply README.md]: updated!

**Comment by qkrybin** [comment]
@xellos00 

Are logos below the contact not shown properly in the dark mode?
image

For me, it appeared like this.
image

**Comment by xellos00** [comment]
> @xellos00
> 
> Are logos below the contact not shown properly in the dark mode? image
> 
> For me, it appeared like this. image

I think you did that in purpose. 
That's more like it. isn't it better? 
Can you ask someone to which one is cool? 
if someone says okay with it. I think I am good with it. ^^

File: README.md (hunk 5 above)
Comment (skonhwang) [reply README.md]: @qkrybin , Thanks 
LGTM!

**Pull request merged**